### PR TITLE
Adding GraphQL World

### PR DESCRIPTION
**Pull request by vning93**
Adding [GraphQL World](https://graphql-world).

Add yourself to the map!

### CODE REVIEW

File: apis.json [hunk 1]
```diff
@@ -121,6 +121,30 @@
...
+      {
+        "description": "GraphiQL",
+        "url": "https://us-west-2.api.scaphold.io/graphql/graphql-world"
+      },
+      {
+        "description": "Docs",
+        "url": "https://graphql-docs.com/docs/?graphqlUrl=https://us-west-2.api.scaphold.io/graphql/graphql-world"
+      },
+      {
+        "description": "Repo",
```
Comment (IvanGoncharov) [inline apis.json]: Can you please change to something like "Homepage", "Site", etc. ?

Comment (vning93) [inline apis.json]: Sure thing, just updated 👍

**Comment by IvanGoncharov** [comment]
Thank you for PR and especially for the map page 👍

**Comment by vning93** [comment]
Of course! 🌍

**Pull request merged**